lacework-agent/templates/daemonset.yaml

```diff
@@ -96,6 +96,9 @@ spec:
             readOnly: true
           - name: podinfo
             mountPath: /etc/podinfo
+{{- if kindIs "string" .Values.laceworkConfig.serviceAccountName }}
+      serviceAccountName: {{ .Values.laceworkConfig.serviceAccountName | quote }}
+{{- end}}
```

Comment:
@cirego Is this documented anywhere?

Why isn't there a corresponding template to create a service account with this name?

I notice that by default, Lacework uses the default service account, which is something that CIS Benchmark recommends against.

However, if I specify serviceAccountName, then I need to create that service account myself.

Reply:
Hi @joebowbeer, no, we have not documented this outside of Lacework as this functionality was only required for OpenShift. As for why we didn't add this functionality before? I commented in more detail on #105.

If you would like us to add a flag to auto-create the Service Account, even for agents that don't access the K8s API Server, please let us know!